### DISCUSSION TITLE
Is it possible to change floppies while running?

### DISCUSSION THREAD

**Discussion by robdaemon** [udo-munk/z80pack#450] [Q&A] [answered]
I'm enjoying z80pack under Linux x86-64 and a RP2040 GEEK, but I was wondering if it's possible to change floppies while it's running, or do I need to `BYE` and swap them out?

**Comment by udo-munk** [comment, marked as answer]
Yes and no. On the RP2040 systems you really have to reset the device to get back into the configuration menu, that lets you change disks then.
z80pack cpmsim opens the disk files once and keeps them open all the time, so it is not possible to change disks while some OS is running on them. This was intentionally implemented to get highest possible speed.
z80pack IMSAI 8080 and Cromemco Z1 have implemented a disk manager, that allows to change disks. You can do that either from the Web GUI, or from a second Linux terminal by changing the links in the disks directory. The Altair 8800 has no Web GUI, but also here disks can be changed by changing the links. Not sure right now about the Intel MDS system and the Mostek box.
And be careful with that, the OS's need a warm boot to re-log disks after changed, if one forgets this the disk gets corrupted.
